lib/crypto/tasks.py

```diff
@@ -264,6 +268,11 @@ def file_supports_firefox(version):
         to_unsign = version.files.filter(ff_version_filter,
```

Comment:
The new `if` doesn't use `to_unsign` though so if it runs this work is un-done.

It seems to me that the `if` should either be something like `to_unsign = to_unsign.filter(...)` or the two cases are mutually exclusive and should be two different cases of the `if`.